aws-py-oidc-provider-pulumi-cloud/README.md

```diff
@@ -0,0 +1,95 @@
+# Provisioning an OIDC Provider in AWS for Pulumi Cloud
+
+This example will create OIDC configuration between Pulumi Cloud and AWS, specifically demonstrating connectivity with [Pulumi ESC](https://www.pulumi.com/docs/pulumi-cloud/esc/). The program automates the process detailed in the AWS documentation for the following activities:
+
+- [Obtaining the thumbprint for an OpenID Connect Identity Provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html)
+- [Creating an OpenID Connect Identity Provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html)
+
+## Prerequisites
+
+* [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+* [Configure Pulumi to Use AWS](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/) (if your AWS CLI is configured, no further changes are required)
+
```

Comment:
Add `git` as a prereq

Reply:
and Python

Reply:
> Add `git` as a prereq

Pushing back a bit on this one. Are we going to add this as a prereq in all of our existing examples and any other existing content we have that uses code from Github?  At the very least I think Git as a pre-requisite should go on the main repo README rather than listed individually in each example.

Reply:
I'm OK with omitting `git` if you feel strongly; your point is certainly valid.